spec/requests/api/v1/court_hearings_controller_create_spec.rb

```diff
@@ -59,6 +59,14 @@
 
         expect(move.court_hearings).to include(court_hearing)
       end
+
+      it 'creates the court hearing in Nomis' do
+        allow(CourtHearings::CreateInNomis).to receive(:call)
```

Comment:
You'll want to hoist this to the top of the context for all specs that create a move.

Reply:
👍